### ISSUE TITLE
Domain and range of gist:geoContains, if correct, should be simply gist:Place

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#334]
Currently:

```
	<rdf:Description rdf:about="&gist;geoContains">
		<rdf:type rdf:resource="&owl;ObjectProperty"/>
		<rdf:type rdf:resource="&owl;TransitiveProperty"/>
		<rdfs:label rdf:datatype="&xsd;string">Geo Contains</rdfs:label>
		<rdfs:comment rdf:datatype="&xsd;string">Transitive version of geoDirectlyContains</rdfs:comment>
		<rdfs:domain>
			<owl:Class>
				<owl:unionOf rdf:parseType="Collection">
					<rdf:Description rdf:about="&gist;GeoPoint">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRegion">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRoute">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoSegment">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoVolume">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;Landmark">
				</owl:unionOf>
			</owl:Class>
		</rdfs:domain>
		<rdfs:range>
			<owl:Class>
				<owl:unionOf rdf:parseType="Collection">
					<rdf:Description rdf:about="&gist;GeoPoint">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRegion">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRoute">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoSegment">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoVolume">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;Landmark">
					</rdf:Description>
				</owl:unionOf>
			</owl:Class>
		</rdfs:range>
	</rdf:Description>
```

and gist:Place is: 
```
	<owl:Class rdf:about="&gist;Place">
		<rdfs:label rdf:datatype="&xsd;string">Place</rdfs:label>
		<rdfs:comment rdf:datatype="&xsd;string">Union of all the geo classes</rdfs:comment>
		<owl:equivalentClass>
			<owl:Class>
				<owl:unionOf rdf:parseType="Collection">
					<rdf:Description rdf:about="&gist;GeoPoint">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRegion">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoRoute">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoSegment">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;GeoVolume">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;Landmark">
					</rdf:Description>
				</owl:unionOf>
			</owl:Class>
		</owl:equivalentClass>
	</owl:Class>
```

Whether or not we like these closed union classes - see issues #322 and #110 - if this one is kept the domain and range assertions in geoContains should be just gist:Place. See also #64 - these must be some copy-paste errors. Fix these two issues together.

I would prefer to use subclass rather than equivalence relations in all three cases.

**Comment by rjyounes** [comment]
Assigning to @uscholdm to fix along with #64 and submit in a single PR.

**Comment by rjyounes** [comment]
Fixed by #363
